features/node/seccomp.feature

```diff
@@ -16,3 +16,43 @@ Feature: Seccomp
     And the output should not contain:
       | 2 |
 
+  # @author jhou@redhat.com
+  # @case_id OCP-32065
+  @admin
+  Scenario: Using Secure Computing Profiles with Pod Annotations
+    # Create custom machine config that contains the seccomp
+    Given I switch to cluster admin pseudo user
+    And I obtain test data file "node/machineconfig_nostat.yaml"
+    When I run the :create admin command with:
+       | f | machineconfig_nostat.yaml |
```

Comment:
Indention is two white spaces. 3 white spaces used here.

Reply:
Fixed that  👍